ghost/core/core/shared/labs.js

```diff
@@ -34,7 +34,8 @@ const GA_FEATURES = [
 const PUBLIC_BETA_FEATURES = [
     'superEditors',
     'editorExcerpt',
-    'additionalPaymentMethods'
+    'additionalPaymentMethods',
+    'themeTranslation'
```

Comment:
We don't tend to put features directly into public beta, there's a surprising number of people who will check everything available without knowing why or what they should be checking.

Would rolling this out as a private beta (hidden but can still be enabled on a per-site basis) first be problematic? It would let us test and gain confidence in the feature design + code before making it more widely available.

Reply:
No objections.  Self-hosters can enable it, right?  Would someone on Pro who wanted to test it just email support?